catalyst/contrib/nn/modules/pooling.py

```diff
@@ -1,5 +1,7 @@
 # flake8: noqa
 # @TODO: code formatting issue for 20.07 release
```

Comment:
```suggestion
```

Reply:
Do I need to follow a specific codestyle ? Because I used the catalyst-codestyle to check and make the codestyles. Or will this PR be closed ??

Reply:
just let me show u the difference :)

Reply:
because of that comment, this file was excluded from our code style checks

Reply:
I will remove those comments and update PR so that the conflict is resolved. I hope it is okay?